### PR TITLE
Fb industrial auth

### PR DESCRIPTION
**Pull request by FadiBaker92**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
+  before_action :is_an_authorized_user, only: [:destroy, :create]
```
Comment (APatel-AI) [inline app/controllers/comments_controller.rb:3]: Nice job on adding an authorization check to make sure that only authorized users can create and destroy comments.

File: app/controllers/comments_controller.rb
Comment (APatel-AI) [inline app/controllers/comments_controller.rb:1]: Great use of functions.

File: app/controllers/photos_controller.rb
Comment (APatel-AI) [inline app/controllers/photos_controller.rb:1]: The CRUD functions are well structured and look correct.

**Review by APatel-AI — APPROVED** [review]
Great Job Fadi!!.

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job, Fadi!

**Pull request merged**